contracts/interfaces/draft-IERC4337.sol

```diff
@@ -84,7 +98,11 @@ interface IEntryPointNonces {
 }
 
 /**
- * @dev Handle stake management for accounts.
+ * @dev Handle stake management for entities (i.e. accounts, paymasters, factories).
+ *
+ * The EntryPoint must implement the following API to let entities like paymasters have a stake,
+ * and thus have more flexibility in their storage access
```

Comment:
The writing is taken directly from the ERC-4337, this description is where the functions for this interface are defined.

It has to do with storage accesses because entities can manage an stake in the Entrypoint to have permissions to access storage that violates validation rules